src/components/projects.tsx

```
@@ -94,11 +93,12 @@ export default function ProjectPage({ params }: { params: { id: string } }) {
         </CardContent>
       </Card>
       <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
-        <Card>
-          <CardContent className="pt-6">
-            <PaymentElementComp />
-          </CardContent>
-        </Card>
+             <Donate 
```

ESLint

[error] Expected indentation of 8 spaces but found 13.

(style/indent)